lib.nix

```diff
@@ -0,0 +1,19 @@
+let
+  # Allow overriding pinned nixpkgs for debugging purposes via cardano_pkgs
+  fetchNixPkgs = let try = builtins.tryEval <cardano_pkgs>;
```

Comment:
I think we can get rid of this conditional as it was never used. Also `maybeEnv` :)